app/build.gradle

```diff
@@ -232,11 +232,20 @@ android {
     }
 
     signingConfigs {
-        release {
-            storeFile file(new File(releaseRepoRoot, 'keys/org.projectbuendia.client.jks'))
-            storePassword ''
-            keyAlias 'org.projectbuendia.client'
-            keyPassword ''
+        if (System.getenv("CI")) {
+            release {
+                keyAlias 'buendia'
+                keyPassword System.getenv('ANDROID_KEYSTORE_PASSWORD')
+                storeFile file(System.getenv('ANDROID_KEYSTORE_FILE'))
+                storePassword System.getenv('ANDROID_KEYSTORE_PASSWORD')
+            }
+        } else {
+            release {
```

Comment:
There's no "release" repo anymore.  Let's change the path to a local location for `zestybuendia.jks` (maybe the root of the `client` repo or nearby?) and use the same `keyAlias` as above so that we can build compatible APKs locally.

Reply:
Fixed